fixtures/issue65-op.js

```diff
@@ -1 +0,0 @@
-var a = 0 + 1;
```

Comment:
Any reason to remove this one?

Reply:
This was was recently added by mistake - it was meant only to be part of the local exploration for #65 (which is wontfix)